requirements/prod.txt

```diff
@@ -52,13 +52,13 @@ cryptography==3.4.7
     #   jwcrypto
     #   pyjwt
     #   sfdo-template-helpers
-cumulusci==3.35.0
+cumulusci==3.35.0.dev0
```

Comment:
Looks like you haven't re-compiled since changing the cumulusci version in prod.in